docs/src/main/paradox/stream/operators/Source-or-Flow/mapWithResource.md

```diff
@@ -11,6 +11,10 @@ Map elements with the help of a resource that can be opened, transform each elem
 2. `f`: Transform each element inputs with the help of resource.
 3. `close`: Close the resource, invoked on end of stream or if the stream fails, optionally outputting a last element.
 
+@apidoc[Flow.mapWithResource](Flow) { scala="#mapWithResource%5BS%20%3C%3A%20AutoCloseable%2C%20T%5D%28create%3A%20%28%29%20%3D%3E%20S%2C%20f%3A%20%28S%2C%20Out%29%20%3D%3E%20T%29%3A%20Repr%5BT%5D" java="#mapWithResource(org.apache.pekko.japi.function.Creator,org.apache.pekko.japi.function.Function2)" }
```

Comment:
```suggestion
@apidoc[Flow.mapWithResource](Flow) { scala="#mapWithResource[S%3C:AutoCloseable,T](create:()=%3ES,f:(S,Out)=%3ET):Repr[T]" java="#mapWithResource(org.apache.pekko.japi.function.Creator,org.apache.pekko.japi.function.Function2)" }
```

Reply:
This approach may help maintain

Reply:
decode ok: `#mapWithResource[S<:AutoCloseable,T](create:()=>S,f:(S,Out)=>T)`

Reply:
✅ updated!